components/automate-ui/src/app/entities/layout/layout-sidebar.service.ts

```diff
@@ -254,17 +254,15 @@ export class LayoutSidebarService {
               this.setVisibleValuesToObservables(menuItem);
             });
           }
-          menuGroup.visible$ = menuGroup.visible$ ? menuGroup.visible$ : new BehaviorSubject(true);
+          menuGroup.visible$ = menuGroup.visible$ || new BehaviorSubject(true);
```

Comment:
I don't follow. If `menuGroup.visible$` is `true`, this will set it to `true`. If it's `false`, this will also set it to `true`. What am I missing?

Reply:
`menuGroup.visible$` is an `Observable<boolean>`, not `boolean`. This is setting the observable to a "fixed" observable of 'true' in the event that the observable is not defined. (It is only defined for things where it needs to be able to vary (is it IAM v2?, is it a feature flag?, and that ilk).